x-pack/plugins/lens/public/metric_visualization/metric_config_panel/title_aize_option.tsx

```diff
@@ -0,0 +1,58 @@
+/*
```

Comment:
This file is currently named `title_aize_option.tsx`. I believe it's meant to be `title_size_option.tsx`.